jenkins/Jenkinsfile

```diff
@@ -119,6 +119,37 @@ pipeline {
                         sh 'docker logout quay.io'
                     }
                 }
+                stage('Upload-PR-to-SonarQube') {
+                    agent { label 'docker' }
+                    environment {
+                        SONAR_HOST_URL = 'https://sonarqube.corp.redhat.com'
+                    }
+                    steps {
+                        sh 'sudo chown -R jenkins:jenkins $WORKSPACE'
+                        checkout scm
+                        sh 'docker login -u "$CANDLEPIN_QUAY_BOT_USR" -p "$CANDLEPIN_QUAY_BOT_PSW" quay.io'
+                        sh 'sh jenkins/upload-on-sonarqube.sh'
+                        sh 'docker logout quay.io'
+                    }
+                }
+                stage('Upload-Branch-to-SonarQube') {
+                    agent { label 'docker' }
+                    environment {
+                        SONAR_HOST_URL = 'https://sonarqube.corp.redhat.com'
```

Comment:
Please remove the url from the code and the commit message, and use a Jenkins secret to pass this in

Reply:
Ack. I missed this configuration.